.travis/utils.sh

```diff
@@ -135,15 +135,33 @@ function lsr_compare_pythons() {
     $(lsr_get_python_version $1) $2 $(lsr_get_python_version $3)
 }
 
+##
+# lsr_get_system_python
+#
+# Return the system python, or /usr/bin/python3 if nothing
+# else can be found in a standard location.
+function lsr_get_system_python() {
+  local syspython=$(command -pv python3)
+  if [[ -z "$syspython" ]]; then
+    syspython=$(command -pv python)
+  fi
+  if [[ -z "$syspython" ]]; then
+    syspython=$(command -pv python2)
+  fi
+  echo ${syspython:-/usr/bin/python3}
+}
+
 ##
 # lsr_venv_python_matches_system_python [$1] [$2]
 #
 #   $1 - command or full path to venv Python interpreter (default: python)
 #   $2 - command or full path to the system Python interpreter
-#        (default: /usr/bin/python3)
+#        (default: /usr/bin/python3 or /usr/bin/python or /usr/bin/python2)
```

Comment:
This comment about the defaults is not accurate, since it checks python3, python and python2 before falling back to /usr/bin/python3 and they could be in different locations than /usr/bin - I believe it is actually the case for Travis that they are not in /usr/bin.

```suggestion
#        (default: system python as determined by lsr_get_system_python())
```